charmcraft/config.py

```diff
@@ -105,6 +115,43 @@ def validate_relative_path(cls, value: str) -> str:
         return value
 
 
+class AttributeName(CustomStrictStr):
+    """Constrainted string which must be the name of an attribute from linters.CHECKERS."""
+
+    @classmethod
+    def validate_relative_path(cls, value: str) -> str:
+        """Validate attribute name."""
```

Comment:
```suggestion
    def validate_name_exists(cls, value: str) -> str:
        """Validate that the attribute checker exists."""
```

Reply:
Ja, silly c&p, good catch! I changed it to `custom_validate`, as it's the generic custom validation called from `CustomStrictStr`.